src/api/.env.example

```
@@ -0,0 +1 @@
+DB_DSN="root:rootpass123@tcp(localhost:3306)/devdb?parseTime=true"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)